corehq/apps/reports/standard/monitoring.py

```diff
@@ -1428,9 +1428,11 @@ def _rows_by_user(self, report_data):
         last_form_by_user = self.es_last_submissions()
         for user in self.users_to_iterate:
             active_cases = int(report_data.active_cases_by_owner.get(user["user_id"].lower(), 0)) + \
-                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]])
+                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]]) + \
```

Comment:
I left the location cases out of the group rows because I wasnt sure what proper behavior should be. If the row shows the number of cases members of that group have access to, it should include that. If it shows number of cases owned by that group, it probably shouldn't. Since I believe that this doesnt include cases owned by other groups the users are in, only those owned by users and the groups in the report, it felt weird to include those owned by the locations the users are in. Happy to change it if you disagree or clarify if that was not explained well.